packages/airnode-protocol/contracts/utils/SelfServeRrpBeaconServerWhitelister.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../rrp/requesters/RrpBeaconServer.sol";
+import "./interfaces/ISelfServeRrpBeaconServerWhitelister.sol";
+
+/// @title Contract that allows to whitlist readers on the RrpBeaconServer
+/// @dev The SelfServeRrpBeaconServerWhitelister contract has the WhitelistExpirationSetterRole
+/// and the IndefiniteWhitelisterRole of the RrpBeaconServer contract. The deployer of this contract
+/// can specify the beaconIds for which readers can whitelist themselves. The deployer (also the owner)
+/// can also add new beaconIds later for readers to self whitelist themselves.
+contract SelfServeRrpBeaconServerWhitelister is
+    Ownable,
```

Comment:
The point of initializing the parameters in the constructor was that this contract was going to be immutable / we wouldn't need to worry about who the owner is and if the wallet would get compromised. So this contract should either be ownable and not do anything in the constructor, or not be ownable and do everything in the constructor and never again.

Reply:
The owner of this contract is going to be the OwnableCallForwarder after deployment, who is also the manager of the `RequesterAuthorizerWithManager`. So I think its fine for this to be a ownable contract because if that is compromised then the `RequesterAuthorizer` is compromised which is a much bigger issue.

Yes, I agree probably better to remove from the constructor then